go/libraries/doltcore/sqle/kvexec/lookup_join.go

```diff
@@ -268,9 +282,32 @@ func newLookupKeyMapping(ctx context.Context, sourceSch schema.Schema, tgtKeyDes
 		targetKb:   val.NewTupleBuilder(tgtKeyDesc, ns),
 		ns:         ns,
 		pool:       ns.Pool(),
+		keyExprs:   keyExprs,
+		idxColTyps: typs,
 	}, nil
 }
 
+// convertLiteralKeyValue converts a literal expression value to the appropriate type for the reference column
+// in a key lookup
+func convertLiteralKeyValue(ctx *sql.Context, colTyp sql.ColumnExpressionType, val *expression.Literal) (any, sql.ConvertInRange, error) {
```

Comment:
This looks suspiciously similar to `convertLookupKey` in `GMS/sql/plan/indexed_table_access.go`